lib/plugins/aws/package/compile/events/stream.js

```diff
@@ -233,6 +237,12 @@ class AwsCompileStreamEvents {
                 event.stream.bisectBatchOnFunctionError;
             }
 
+            if (event.stream.functionResponseType != null) {
+              streamResource.Properties.FunctionResponseTypes = [];
```

Comment:
Let's maybe go with `streamResource.Properties.FunctionResponseType = [event.stream.functionResponseType];` to be more succinct here

Reply:
Sure thing. Will be pushing the changes soon

Reply:
@pgrzesik Done :)